packages/edit-site/src/components/post-edit/index.js

```diff
@@ -0,0 +1,65 @@
+/**
+ * WordPress dependencies
+ */
+import { __ } from '@wordpress/i18n';
+import { DataForm } from '@wordpress/dataviews';
+import { useDispatch, useSelect } from '@wordpress/data';
+import { store as coreDataStore } from '@wordpress/core-data';
+import { Button } from '@wordpress/components';
+import { useState, useMemo } from '@wordpress/element';
+
+/**
+ * Internal dependencies
+ */
+import Page from '../page';
+import usePostFields from '../post-fields';
+
+export function PostEdit( { postType, postId } ) {
```

Comment:
We have `PostList` to render dataviews for a given post type.
This new component `PostEdit` is meant to edit the "form" or "inspector panel" of a given post type.